server/resourcemanager/resource_manager.go

```diff
@@ -147,6 +147,9 @@ func (m *manager[T]) list(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// TODO: the name "count" can be misleading when using pagination.
```

Comment:
this field hold the `total count`, not the page count. I see how this can be confusing. feel free to rename the field

Reply:
No problem, I'll do this in another PR!